mock-core-configs/etc/mock/templates/centos-stream-8.tpl

```diff
@@ -57,63 +57,63 @@ gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Officia
 
 [Stream-base-debuginfo]
 name=CentOS-Stream - Debuginfo
-baseurl=http://debuginfo.centos.org/8-stream/$basearch/
+baseurl=http://debuginfo.centos.org/stream-8/$basearch/
```

Comment:
These repositories give me error 404.  Should they stay in the `8-stream` format?

Reply:
We haven't started publishing debuginfo packages for CS8 yet.  An 8-stream directory exists, but it's empty.  That will likely be the correct location once we do start publishing debuginfo packages.

Reply:
In fact all the baseurls should stick with `8-stream` for now.

Reply:
Yep, that's what I meant.